core/roles/ingenieroANT.sqf

```diff
@@ -1,44 +0,0 @@
-/*******************************************************************************
-                          Realizado por |ArgA|Ignacio
-						  Rol por       |ArgA|Veck|Cbo¹
-*******************************************************************************/
-
-params [["_unit", player], ["_helmet", "arga_cas_combate_bosque_marpat"], ["_backPack", "arga_m_petate_bosque_marpat"], ["_backPackLittle", "arga_m_asalto_bosque_marpat"]];
```

Comment:
did you delete this role from the role list? So no one can pick it

Reply:
I'm not seeing the option in the list, what is this file? Something that we forgot to delete?

Reply:
It was a backup that was never deleted